### DISCUSSION TITLE
BioProject with NTM Data for testing

### DISCUSSION THREAD

**Discussion by harrismia2** [jodyphelan/NTM-Profiler#43] [Ideas]
Hello @jodyphelan , 

  I found this bio project:[PRJNA319839](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA319839) which has WGS data for a large number of NTMs.  This might be good for testing purposes.  Thanks for creating this tool!  There is not a lot out there for NTM sequence analysis.

**Comment by jodyphelan** [comment]
Thanks @harrismia2 this will be useful to test out the speciation function, I'll take a look.
